tests/cases/standalone/common/alter/alter_table_options.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE ato(i INTEGER, j TIMESTAMP TIME INDEX);
+
+ALTER TABLE ato MODIFY 'ttl'='1d';
+
+SHOW CREATE TABLE ato;
+
+ALTER TABLE ato MODIFY 'ttl'='2d';
```

Comment:
Looks like it's not the syntax we discussed before.https://github.com/GreptimeTeam/greptimedb/issues/1084

Reply:
> Looks like it's not the syntax we discussed before.#1084

In Ansi-SQL, SET keyword is often used as setting session-wise variables. Databases like Clickhouse use `MODIFY` keyword to change TTL parameters.

Reply:
I got it, but I prefer `MODIFY OPTIONS ttl='2d'` like that, it represents that you want to modify the option just like `MODIFY COLUMN`, which means you want to modify the column options.

Reply:
> I got it, but I prefer `MODIFY OPTIONS ttl='2d'` like that, it represents that you want to modify the option just like `MODIFY COLUMN`, which means you want to modify the column options.

OK. That will also make parser kind of easier.

Reply:
fixed in f89b532dd1